src/video_core/engines/maxwell_3d.h

```diff
@@ -1194,12 +1242,23 @@ class Maxwell3D final {
 
     Upload::State upload_state;
 
+    static constexpr u32 null_cb_data = 0xFFFFFFFF;
+    struct {
+        std::array<std::array<u32, 0x4000>, 16> buff;
```

Comment:
```suggestion
        std::array<std::array<u32, 0x4000>, 16> buffer;
```

There's no need to mince words here.